src/Modules/Visualization/ShowField.cc

```diff
@@ -430,6 +430,96 @@ void GeometryBuilder::renderFaces(
 
 
 
+inline static void writeAtributeToVBO(const Point& point, spire::VarBuffer* vboBuffer)
+{
+  vboBuffer->write(static_cast<float>(point.x()));
+  vboBuffer->write(static_cast<float>(point.y()));
+  vboBuffer->write(static_cast<float>(point.z()));
+}
+
+inline static void writeAtributeToVBO(const Vector& vector, spire::VarBuffer* vboBuffer)
+{
+  vboBuffer->write(static_cast<float>(vector.x()));
+  vboBuffer->write(static_cast<float>(vector.y()));
+  vboBuffer->write(static_cast<float>(vector.z()));
+}
+
+inline static void writeAtributeToVBO(const ColorRGB& color, spire::VarBuffer* vboBuffer)
+{
+  vboBuffer->write(static_cast<float>(color.r()));
+  vboBuffer->write(static_cast<float>(color.g()));
+  vboBuffer->write(static_cast<float>(color.b()));
+  vboBuffer->write(static_cast<float>(1.0f));
+}
+
+inline static void writeAtributeToVBO(const glm::vec2& coords, spire::VarBuffer* vboBuffer)
+{
+  vboBuffer->write(static_cast<float>(coords.x));
```

Comment:
See above--although, the function I wrote ought to be a member function of VarBuffer